modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt

```diff
@@ -585,42 +590,88 @@ class InstructionsTest {
     @WithIroha(DefaultGenesis::class)
     fun `register peer instruction committed`(): Unit = runBlocking {
         val address = "127.0.0.1:1338"
-        val payload = "ed012076cd895028f2d9d520d6534abd78def38734b658f9400c31b3212ed42a423ee3".fromHex()
+        val payload = "76cd895028f2d9d520d6534abd78def38734b658f9400c31b3212ed42a423ee3".fromHex()
 
         registerPeer(address, payload)
         assertTrue(isPeerAvailable(address, payload))
     }
 
-    // TODO: unregister peer test (https://github.com/hyperledger/iroha/issues/1726)
+//    @Test
```

Comment:
why are those commented then?

Reply:
It doesn't work until task iroha-java/209 is done